### ISSUE TITLE
Will the search plugins work?

### ISSUE THREAD

**Issue by worldsdream** [minnyres/qbittorrent-windows-arm64#1]
In qBittorent we have native support for search plugins. I know that it will install python 3 and when you perform a search, it will load op python in the back-end.

Will this also work in your version? Thank you.

**Comment by minnyres** [comment]
Yes it works. I would recommend you install python manually before use search plugins in qBittorrent.

**Comment by worldsdream** [comment]
That’s great. 

I will install Python for arm64 manually from:

https://www.python.org/downloads/windows/

**Comment by DinuruSeniya** [comment]
Hi, you can close this issue @minnyres
